internal/telemetry/event.go

```diff
@@ -82,13 +86,13 @@ func withChoice(c string) eventOpt {
 }
 
 func sanitizeSelectOption(v string) string {
-	parenthesesRegex := regexp.MustCompile(`^.*\(([^\(\)]*)\)$`)
+	parenthesesRegex := regexp.MustCompile(`^.*\(([^()]*)\)$`)
 
 	return parenthesesRegex.ReplaceAllString(v, "$1")
 }
 
 func sanitizePrompt(q string) string {
-	bracketsRegex := regexp.MustCompile(`\[[^\]\[]*\]`)
+	bracketsRegex := regexp.MustCompile(`\[[^]\[]*]`)
```

Comment:
unrelated change?

Reply:
drive by, intellij was complaining of unnecessary scape